doc/manual/manual.tex

```diff
@@ -2868,6 +2868,38 @@ \subsubsection{Adjusting solver tolerances} At the heart of every time step
 point where the output quantities start to change significantly. This is the
 point where you will want to stop.
 
+\subsubsection{Adjusting solver preconditioner tolerances} To solve the Stokes 
+equations it is necessary to lower the condition number of the
+Stokes matrix by preconditioning  it. In \aspect{} a right preconditioner $Y^{-1} = 
+\begin{pmatrix}
+\widetilde{A^{-1}} & -\widetilde{A^{-1}}B^{T}\widetilde{S^{-1}} \\
+0 & \widetilde{S^{-1}}
+\end{pmatrix}$ is used to precondition the system, where $\widetilde{A^{-1}}$ is 
+the approximate inverse of the A block and $\widetilde{S^{-1}}$ is the approximate 
+inverse of the Schur complement matrix. Matrix $\widetilde{A^{-1}}$ and 
+$\widetilde{S^{-1}}$ are calculated through a CG solve, which requires a tolerance 
+to be set. In comparison with the solver tolerances of the previous section, these 
+parameters are relatively safe to use, since they only change the preconditioner, 
+but can speed up or slow down the solving of the Stokes system considerably. 
+
+In practice $\widetilde{A^{-1}}$ takes by far the most time to compute, but is 
+also very important in conditioning the system. The computation of 
```

Comment:
Please make computation a bit more precise, either "accuracy of the computation" or "residual tolerance", something like this.